src/app/general-style/general-style.component.html

```diff
@@ -0,0 +1,5 @@
+<!-- <p>general-style-component</p> -->
+<label
+  >Name--{{ prop
+  }}<input formControlName="forControlAcc" [(ngModel)]="value" type="text"
```

Comment:
we do not need formControlName and ngModel on the same input
read more documentation about ControlValueAccessor interface